fastlane/spec/actions_specs/download_dsyms_spec.rb

```diff
@@ -74,6 +75,36 @@
         end
       end
 
+      context 'when version is live' do
+        before do
+          # live
+          allow(app).to receive(:live_version).and_return(live_version)
```

Comment:
full disclosure: I was not sure how to adapt the tests with a live_version. I hope this is correct 😅